### PR TITLE
[13.0][FIX] stock_buffer_capacity_limit: only consider qty on buffered location v2

### PR DESCRIPTION
**Pull request by TDu**
This is a temporary duplicate of #113 Just to fix the linting

### CODE REVIEW

**Review by sebalix — APPROVED** [review]

File: stock_buffer_capacity_limit/models/stock_buffer.py [hunk 1]
```diff
@@ -16,7 +16,12 @@ class StockBuffer(models.Model):
         "execution alerts.",
     )
 
-    @api.depends("storage_capacity_limit")
+    @api.depends(
+        "storage_capacity_limit",
+        "product_location_qty",
```
Comment (simahawk) [inline stock_buffer_capacity_limit/models/stock_buffer.py]: `product_location_qty` AFAIS is not stored, you cannot use it here

Comment (sebalix) [reply stock_buffer_capacity_limit/models/stock_buffer.py]: You are sure? I thought we could, at least to trigger the computation from the UI like would do an onchange. Maybe I'm wrong.

Comment (simahawk) [reply stock_buffer_capacity_limit/models/stock_buffer.py]: normally I get a warning from server logs complaining the the field is not stored and, as such, is useless to hook to its change. Am I missing anything?

Comment (TDu) [reply stock_buffer_capacity_limit/models/stock_buffer.py]: It is not stored indeed.
But I did not know that rule either.
Removed from the depends...

Comment (simahawk) [reply stock_buffer_capacity_limit/models/stock_buffer.py]: bah I cannot find any reference in the code...but https://github.com/odoo/odoo/issues/37468
It seems it happens only if the field has no depends specified?
`_compute_product_available_qty` has no depends....maybe it needs a fix as well... who wins? :smile:

Comment (mmequignon) [reply stock_buffer_capacity_limit/models/stock_buffer.py]: you do

**Comment by LoisRForgeFlow** [comment]
#113 has been merged, I think this is not needed anymore. I'm closing it, please reopen if needed.

**Pull request closed without merging**